test/suites/drift/suite_test.go

```diff
@@ -65,8 +65,13 @@ var _ = Describe("Drift", Label("AWS"), func() {
 			"featureGates.driftEnabled": "true",
 		})
 		// choose an old static image
+		parameters, err := env.SSMAPI.GetParametersByPath(&ssm.GetParametersByPathInput{
+			Path:      awssdk.String("/aws/service/eks/optimized-ami/1.23/amazon-linux-2"),
```

Comment:
Can you also parameterize the kubernetes version to use the one that's discovered by the environment?

Reply:
Additionally, you could consider putting this in a function to get the latest-nth version with a `n int` parameter.